CHANGES.md

```diff
@@ -1,12 +1,26 @@
 # Release Notes
 
 ## Known issues
-- Many decoders will panic on malicous input. In most cases, this is caused by
-  not enforcing memory limits, though other panics have been seen from fuzzing.
+- Many decoders will panic on malicious input.
 - The color space information of pixels is not clearly communicated.
 
 ## Changes
 
+### Version 0.25.3
+
+Features:
+ - Added support for orientation metadata, so that e.g. smartphone camera images could be displayed correctly:
+   - Added `ImageDecoder::orientation()` and implemented orientation metadata extraction for JPEG, WebP and TIFF formats
+   - Added `DynamicImage::apply_orientation()` to apply the orientation to an image
+ - Added support for extracting Exif metadata from images via `ImageDecoder::exif_metadata()`, and implemented it for JPEG and WebP formats
+ - Added `ImageEncoder::set_icc_profile()` and implemented it for WebP format. Pull requests with implementations for other formats are welcome.
+ - Added `DynamicImage::fast_blur()` for a linear-time approximation of Gaussian blur, which is much faster at larger blur radii
+
+Bug fixes:
+ - Fixed some APNG images being decoded incorrectly
+ - Fixed the iterator over animated WebP frames to return `None` instead of an error when the end of the animation is reached
+ - Disabling the `rayon` feature now also disables multi-threading in `ravif` and actually removes `rayon` from the dependency tree
```

Comment:
The "actually removes rayon from the dependency tree" part isn't true yet. https://github.com/johannesvollmer/exrs/pull/241 also needs to land first (tracked by https://github.com/image-rs/image/issues/2028)

Reply:
And actually, don't we need to change our Cargo.toml to enable ravif's threading feature when our rayon feature _is_ enabled?

Reply:
We do, and in fact the latest `ravif` with `default-features=false`  will not enable rayon by default anymore, so this release is needed just to preserve the previous default behavior when using the latest `ravif`.

I've opened https://github.com/image-rs/image/pull/2348 for it